html-forms/index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>HTML Forms</title>
+</head>
+<body>
+  <h1>Coding Form!</h1>
+  <form>
+    <div>
+      <label for="email">Email</label>
+    </div>
+    <div>
+      <input type="text" name="email" required>
+    </div>
+    <div>
+      <label for="coding">Why is coding cool?</label>
+    </div>
+    <div>
+      <select name="why-coding">
+        <option value="option">Select an option</option>
+        <option value="interesting">Because it's interesting</option>
+        <option value="fun">Because it's fun</option>
+        <option value="powerful">Because it's powerful</option>
+      </select>
+    </div>
+    <div>
+      <label for="language">Favorite Language (choose one)</label>
+    </div>
+    <div>
+      <input type="radio" name="language" value="HTML"> HTML
```

Comment:
so here because there are three inputs, you want to have three labels, and since these are radio inputs, its a good situation to wrap the input with the label. Let me know if you have questions!